sdk/java/pulumi/src/main/java/com/pulumi/provider/internal/properties/PropertyDeserializationException.java

```diff
@@ -0,0 +1,56 @@
+package com.pulumi.provider.internal.properties;
```

Comment:
Not necessarily something we need to fix in this PR, but I'm curios, do we not do copyright/license headers for `java` files in this repository?

Reply:
It's lacking throughout. I added a copyright header to all the new auto API files. But there's no tooling yet to enforce it being there and in a consistent format. Opened https://github.com/pulumi/pulumi-java/issues/1658